airflow/models.py

```diff
@@ -3032,6 +3032,25 @@ def get_task_instances(
     def roots(self):
         return [t for t in self.tasks if not t.downstream_list]
 
+    def topographical_sort(self):
```

Comment:
Should be named topological sort (https://en.wikipedia.org/wiki/Topological_sorting) for consistency